src/_style.scss

```diff
@@ -14,6 +14,22 @@ $microsoft-focus-black: #000;
 $apple-black: #000000;
 $apple-focus-black: $apple-black;
 
+.sr-only {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
+
+.focus-out {
+  position: absolute;
+  padding-left: 17px;
+  opacity: 0.75;
+  z-index: 1;
+}
```

Comment:
What this focus-out class needed for?

Reply:
I have mimicked the functionality of FE. 

- The placeholder is removed
- Label is used in place of placeholder i.e move the Label down above the form field using focus-out class
- Label is moved up when user focus in

Reply:
Got it! Can we add some unit tests if possible to verify .sr-only class is applied on required fields.